plugin/forward/health_test.go

```diff
@@ -43,6 +43,37 @@ func TestHealth(t *testing.T) {
 	}
 }
 
+func TestHealthNoRecursion(t *testing.T) {
```

Comment:
this doesn't test the functionality you've added. The only thing we care about is if the incoming query now has the recursion bit set or not

Reply:
Redid both tests i added. Now checks the RD-flag in the dns-requests.